__tests__/localAgent.test.ts

```diff
@@ -147,6 +152,12 @@ const setup = async (options?: IAgentOptions): Promise<boolean> => {
       new DIDComm(),
       new CredentialIssuer(),
       new SelectiveDisclosure(),
+      new DIDDiscovery({
+        providers: {
+          'alias': new AliasDiscoveryProvider(),
+          'profile': new ProfileDiscoveryProvider(),
+        }
+      }),
```

Comment:
I think it would be nicer to have this be an array.
This would reduce the naming burden on the user.
```suggestion
      new DIDDiscovery([
        new AliasDiscoveryProvider(),
        new ProfileDiscoveryProvider(),
      ]),
```

Reply:
I agree with this